packaging/snapd.mk

```diff
@@ -74,6 +74,7 @@ $(builddir)/snap $(builddir)/snap-seccomp $(builddir)/snapd-apparmor:
 		-buildmode=pie \
 		-ldflags="$(EXTRA_GO_LDFLAGS)" \
 		-mod=vendor \
+		-buildvcs=false \
```

Comment:
Is this relevant now?

Reply:
no, it was only needed on opensuse when I was building an RPM locally, not sure why our tests don't need it but perhaps they use a slightly different artifact

Reply:
I think vcs meta-data is somewhat useful so perhaps we can drop it soon when re-visiting packaging/snapd.mk